.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,109 @@
+name: Zeebe CI (AWS OS)
+
+on:
+  workflow_dispatch: { }
+  workflow_call: { }
```

Comment:
This means the workflow is triggered by either humans or another workflow (which is not visible in this PR). It would be useful to know from where you want to trigger it, for a full review.